kategory-test/src/main/kotlin/kategory/generators/Generators.kt

```diff
@@ -55,3 +55,8 @@ fun genIntPredicate(): Gen<(Int) -> Boolean> =
                     { it % absNum == absNum - 1 })
             )
         }
+
+fun <B> genOption(genB: Gen<B>): Gen<Option<B>> =
```

Comment:
You need to capture genIntSmall().generate() as a field inside the object, else you won't get adequate distributions as it restarts the seed every time.